File: DayTwo/JavaScript/QuestionSix.js


let runAll = (number) => {
    if(isPalindrome(number) && isPrime(number)) return true;
    return false;

}



let isPalindrome = (number) => { 
        let newNumber = String(number);
        let reverse = "";

        for(let count = 0; count < array.length; count++){
            reverse = newNumber.charAt(count) + reverse;        
         }

        let newreverse = Number(reverse);
        if(newreverse == number) return true
    return false
}



let isPrime = (number)=> {
    let count = 0;
    for(let divisor = 0; divisor <= number; divisor++){
        if(number % divisor == 0) count++;
            
    }
    if(count == 2) return true;

  return false;

}


console.log(runAll(1234));

File: DayTwo/Python/question_six.py
def is_palindrome(number):

    if(number == str(number)[::-1]): return True

    return False



def isPrime(number):
    count = 0;
    for divisor in range(number+1):
        if(number % divisor == 0): count+=1;
            
    if(count == 2): return True;

    return False


def run_all(number):
    if(is_palindrome(number) and  isPrime(number)): return True
    return False



print(run_all(1234))
